app/graphql/mutations/profile/edit.rb

```diff
@@ -19,7 +19,7 @@ class Edit < BaseMutation
 
       def resolve(args = {})
         profile = authorized_profile(args)
-        profile.policy_object.update(args.compact.slice(*POLICY_ATTRIBUTES))
+        profile.policy_object.update(args.slice(*POLICY_ATTRIBUTES))
```

Comment:
The `.compact` was there even before. This bug is present since RHICOMPL-742.

Reply:
Yeah, I'm really not sure why we never caught this before :man_shrugging: